internal/scheduler/yunikorn/scheduler.go

```diff
@@ -0,0 +1,148 @@
+package yunikorn
```

Comment:
We can add license comment in every new file, like https://github.com/kubeflow/spark-operator/blob/5972482ca4dbe059914a9b5b70edb98a82dfc0a6/internal/controller/sparkapplication/controller.go#L1-L15.

Reply:
I'll add the license comment to the top of all my new files, but what do you think if in a separate PR we add a license comment check as part of CI, and maybe a Makefile target for checking and adding locally?

Reply:
I think it is a good point to add license check as part of CI. cc @andreyvelich @vara-bonthu @yuchaoran2011 .

Reply:
Created an issue for it https://github.com/kubeflow/spark-operator/issues/2139